src/shogun/statistics/QuadraticTimeMMD.cpp

```diff
@@ -1022,8 +1024,9 @@ SGVector<float64_t> CQuadraticTimeMMD::sample_null_spectrum_DEPRECATED(
 	}
 
 	/* when m=n, return m*MMD^2 instead */
-	if (m==n)
-		null_samples.scale(0.5);
+	if (m==n) {
+		linalg::scale<linalg::Backend::NATIVE>(null_samples, 0.5);
```

Comment:
same as above. Also, a minor styling issue - in Shogun we don't use braces with `if-else` and loops if it is a single liner. For braces we follow the
```c++
if (cond)
{
    // do stuffs
}
```
instead of
```c++
if (cond) {
   // do stuff
}
```